### DISCUSSION TITLE
How do I disable double-click the tab bar to open a new tab?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#7842] [Q&A] [answered]
How do I prevent double-clicking the tab bar from opening a new tab?

None of the `mouse_map` entries in the default `kitty.conf` document this behavior.

**Comment by kovidgoyal** [comment, marked as answer]
You dont, its hard coded.

On Tue, Sep 10, 2024 at 12:51:46AM -0700, salmankhilji wrote:
> How do I prevent double-clicking the tab bar from opening a new tab?
>
> None of the `mouse_map` entries in the default `kitty.conf` document this behavior.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7842
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
